.github/workflows/security.yaml

```diff
@@ -0,0 +1,21 @@
+name: Security Scan
+
+on:
+  push:
+    branches:
+      - main
+
+permissions: write-all
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Trivy vulnerability scanner on postgres image
+        uses: aquasecurity/trivy-action@0.24.0
```

Comment:
use the hash instead of the raw version, e.g. `uses: foo/bar@asdfjkl1243  # 0.24.0`